CRM/Contribute/BAO/Contribution/Utils.php

```diff
@@ -575,12 +575,18 @@ public static function getContributionStatuses($usedFor = 'contribution', $id =
         $statusNamesToUnset = array_merge($statusNamesToUnset, [
           'Cancelled',
           'Failed',
+          'In Progress',
+          'Overdue',
+          'Partially paid',
         ]);
       }
       elseif ($usedFor == 'membership') {
         $statusNamesToUnset = array_merge($statusNamesToUnset, [
           'In Progress',
+          'Cancelled',
+          'Failed',
           'Overdue',
+          'Partially paid',
```

Comment:
@eileenmcnaughton I'm a bit puzzled by this bit of code and the comment.  Why are we unsetting different statuses for participant/membership?

Reply:
After this they should be the same - before this I think it's historical randomness

Reply:
@eileenmcnaughton Ok. So if you update this comment which will be wrong after this PR I'm happy to merge this: https://github.com/civicrm/civicrm-core/pull/15771/files#diff-99d052aa6a966c5a2cc89d05f057d22dR565-R566